xml/System.Numerics/BigInteger.xml

```diff
@@ -6627,12 +6624,85 @@ value Mod 2 = 0
         <Parameter Name="provider" Type="System.IFormatProvider" Index="2" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="style">To be added.</param>
-        <param name="provider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">A read-only span of characters that contains the number to convert.</param>
+        <param name="style">A bitwise combination of the enumeration values that specify the permitted format of <paramref name="value" />.</param>
+        <param name="provider">An object that provides culture-specific formatting information about <paramref name="value" />.</param>
+        <summary>Converts the representation of a number, contained in the provided read-only span of characters, in a specified style to its <see cref="T:System.Numerics.BigInteger" /> equivalent.</summary>
+        <returns>A value that is equivalent to the number specified in the <paramref name="value" /> parameter.</returns>
+        <remarks>
```

Comment:
I copied these remarks from the overload that receives a string in the place of the ReadOnlySpan. I removed the sections that I considered specific for strings, and left others that simply needed "string" to be renamed to either "span" or "`value`".

I did the same with the other long remark for the TryParse equivalent method.